app/controllers/api/v2/application_controller.rb

```diff
@@ -17,7 +17,8 @@ def authenticate
                                                  Settings.authentication.algorithm)
             email = payload['email']
           end
-          @current_user = User.find_by(email: email)
+          # match email addresses case-insensitively
+          @current_user = User.find_by("lower(email) = ?", email.downcase)
```

Comment:
Do we know if this will sidestep the index we have on the `email` column? I haven't checked yet but maybe if we can normalise emails in the db to be lower-case we can avoid running `lower(email)`?

Reply:
Agreed with Jake that we'll look at addressing this later, as we can't yet ensure 100% that all emails entered in the DB are lower-case (because of multiple input sources).

Reply:
Reminder card created: https://trello.com/c/pZx8vqUM/212-dont-break-the-index-when-looking-for-users-in-the-db